src/common/api/saving/remove.js

```diff
@@ -2,15 +2,15 @@ import { request } from '../_request/request'
 
 /* API CALLS - Should return promises
 –––––––––––––––––––––––––––––––––––––––––––––––––– */
-export function removeItem(itemId, actionInfo) {
+export function removeItem(itemId) {
   return request({
     path: 'send/',
     data: {
       actions: [
         {
           action: 'delete',
           item_id: itemId,
-          ...actionInfo
+          type: 'page'
```

Comment:
We don't need to pass in any extra info. This was hardcoded as `type: page` in the previous version of the extension 😂